test/alternator/test_filter_expression.py

```diff
@@ -669,13 +669,13 @@ def test_filter_expression_and_sort_key_condition(test_table_sn_with_data):
 # In particular, test that FilterExpression may inspect attributes which will
 # not be returned by the query, because of the ProjectionExpression.
 # This test reproduces issue #6951.
-def test_filter_expression_and_projection_expression(test_table):
+def test_filter_expression_and_projection_expression(test_table_ss):
```

Comment:
please move all these renamings to a separate patch (can be same PR, if you want).
I'm not sure why you decided to do this right now, but in any case it's not really related to the main patch.